src/apps/relay/prom_server.c

```diff
@@ -35,6 +35,44 @@ prom_counter_t *turn_total_traffic_peer_sentb;
 
 prom_gauge_t *turn_total_allocations;
 
+#if MHD_VERSION >= 0x00097500
```

Comment:
Seems like MHD_Result was introduced in libmicrohttpd 4 years ago in 0.9.71, not 0.9.75
https://github.com/Karlson2k/libmicrohttpd/commit/6347f514aa2388e774d5bf356df8046864e5f73c

Also, this was not in promhttp - so we are not losing anything by not having `#if MHD_VERSION >= 0x00097500`

I suggest to simplify the code and require libmicrohttpd >= 0.9.71
What do you think?

Reply:
Yeah, I think it makes a lot of sense. There's some other code also checking the version of microhttpd, like the error log and epoll flags. Should we remove those ifdefs in this PR or take that another time?

Reply:
Please remove it now - no point in adding and then removing code.
Thanks!

Reply:
The CI is installing libmicrohttpd 0.9.66-1, seemingly due to ubuntu:20.04 being used. If I try to run ubuntu:22.04 instead, I get 0.9.75. 

I think I've seen a few issues and PRs on this topic. I guess the "easiest" thing to do is to update the CI/CD and start requiring newer versions of ubuntu. Otherwise we should just be able to revert my latest commit, again adding support for <0.9.71.

Reply:
Ugh. Let's use `int` return type instead of `MHD_Status` - ugly but should work I hope.

Reply:
I see. Let's do that (revert). Maybe you can make it a little nicer by defining `MHD_Result` as int for older version and keeping the code in its new form
```
#if MHD_VERSION < 0x00097500
define MHD_Result int
#endif

MHD_Result promhttp_handler(....
```

Reply:
Note that it's `enum MHD_Result`. I defined a `MHD_RESULT` that works the way you described. Does that work for you?

Also, I stumbled upon a changelog entry for the change. It recommends using `MHD_VERSION >= 0x00097002`. So I changed that as well. Here's a link to the changelog: https://github.com/Karlson2k/libmicrohttpd/blob/a1085032dc631e387eb8e06b41a382e7dc6dd8eb/ChangeLog#L1055.